examples/table.tape

```diff
@@ -1,19 +1,30 @@
 # This is a vhs script. See https://github.com/charmbracelet/vhs for more info.
 # To run this script, install vhs and run `vhs ./examples/table.tape`
 Output "target/table.gif"
-Set Theme "Aardvark Blue"
-Set Width 1200
-Set Height 800
+Set Width 1600
+Set Height 864
```

Comment:
These gifs end up going in the examples readme, so I like to make them look a bit consistent. Changing the width here might make things look a bit weird (or may be fine).

Can you publish a gif with your changes using `vhs publish` and update the example readme so we can see this in context of what it looks like on GitHub?

Reply:
Looks good imo. I also set the theme as `"Aardvark Blue"` to not break the consistency. You can see it in the last commit.